packages/grid/src/vaadin-grid.d.ts

```diff
@@ -299,29 +299,41 @@ export interface GridEventMap<TItem> extends HTMLElementEventMap, GridCustomEven
- * `row`                  | Row in the internal table
- * `expanded-row`         | Expanded row
- * `selected-row`         | Selected row
- * `details-opened-row`   | Row with details open
- * `odd-row`              | Odd row
- * `first-row`            | The first body row
- * `last-row`             | The last body row
- * `dragstart-row`        | Set on the row for one frame when drag is starting. The value is a number when multiple rows are dragged
- * `dragover-above-row`   | Set on the row when the a row is dragged over above
- * `dragover-below-row`   | Set on the row when the a row is dragged over below
- * `dragover-on-top-row`  | Set on the row when the a row is dragged over on top
- * `drag-disabled-row`    | Set to a row that isn't available for dragging
- * `drop-disabled-row`    | Set to a row that can't be dropped on top of
- * `cell`                 | Cell in the internal table
- * `header-cell`          | Header cell in the internal table
- * `body-cell`            | Body cell in the internal table
- * `footer-cell`          | Footer cell in the internal table
- * `details-cell`         | Row details cell in the internal table
- * `focused-cell`         | Focused cell in the internal table
- * `resize-handle`        | Handle for resizing the columns
- * `reorder-ghost`        | Ghost element of the header cell being dragged
+ * Part name                  | Description
+ * ---------------------------|----------------
+ * `row`                      | Row in the internal table
+ * `expanded-row`             | Expanded row
+ * `selected-row`             | Selected row
+ * `details-opened-row`       | Row with details open
+ * `odd-row`                  | Odd row
+ * `first-row`                | The first body row
+ * `last-row`                 | The last body row
+ * `dragstart-row`            | Set on the row for one frame when drag is starting. The value is a number when multiple rows are dragged
+ * `dragover-above-row`       | Set on the row when the a row is dragged over above
+ * `dragover-below-row`       | Set on the row when the a row is dragged over below
+ * `dragover-on-top-row`      | Set on the row when the a row is dragged over on top
+ * `drag-disabled-row`        | Set to a row that isn't available for dragging
+ * `drop-disabled-row`        | Set to a row that can't be dropped on top of
+ * `cell`                     | Cell in the internal table
+ * `header-cell`              | Header cell in the internal table
+ * `body-cell`                | Body cell in the internal table
+ * `footer-cell`              | Footer cell in the internal table
+ * `details-cell`             | Row details cell in the internal table
+ * `loading-cell`             | Cell in a row that is waiting for data from data provider
+ * `focused-cell`             | Focused cell in the internal table
+ * `selected-cell`            | Cell in a selected row
+ * `expanded-cell`            | Cell in an expanded row
```

Comment:
Should these still have the `-row` in the middle as suggested in https://github.com/vaadin/web-components/pull/5016#discussion_r1023076198?

Reply:
Thanks, will do.

Reply:
Changed to use `-row-cell` for all of those: `loading`, `selected`, `expanded`, `details-opened`.